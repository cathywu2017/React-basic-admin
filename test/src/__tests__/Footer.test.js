import React from 'react';
import {
  cleanup,
  render,
} from '@testing-library/react';
import Footer from '../Footer';

afterEach(cleanup);

it('renders Footer correctly', () => {
  const { container } = render(
    <Footer />,
  );

  expect(container).toMatchSnapshot();
});
