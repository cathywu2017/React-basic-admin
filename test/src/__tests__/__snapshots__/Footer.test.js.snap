// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer correctly 1`] = `
<div>
  <div
    class="ui inverted vertical segment"
  >
    <div
      class="ui center aligned container"
    >
      <p>
        ©中佑資訊有限公司
      </p>
    </div>
  </div>
</div>
`;
